lib/model.js

```diff
@@ -62,6 +62,7 @@ assign(props, {
    *   Analyics Box
    */
   name: { type: 'string', default: 'Local' },
+  color: { type: 'object', default: () => ({}) },
```

Comment:
Why is `color` type `object` and default to a function? My expectation is that it's a simple hex string. And why does it need to be defined twice when extended-model extends the base model?

Reply:
https://github.com/mongodb-js/connection-model/pull/204